### PR TITLE
VueJS admin settings

### LINKED ISSUE #1564: /commands overview in admin settings should be a beautiful grid
as discussed @skjnldsv

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Migrate STUN
  - [x] Validate input
  - [x] Store result
- [x] Migrate TURN
  - [x] Test automatically #1305
- [x] Migrate Signaling
  - [x] Changing secret does not update the config
  - [x] Checkbox for verifying the cert
- [x] Make Commands beautiful #1564
- [x] Migrate Allowed groups

Fix #1564

### CODE REVIEW

File: vue/src/StunServerSettings.vue [hunk 1]
```diff
@@ -22,56 +22,98 @@
...
+		}, 1000)
+	},
 
-			newServer () {
-				this.servers.push('');
-			},
+	methods: {
+		removeServer(index) {
+			this.servers.splice(index, 1);
+			if (this.servers.length === 0) {
```
Comment (skjnldsv) [inline vue/src/StunServerSettings.vue]: Not sure if this is a good idea or not :thinking:

File: vue/src/SignalingServerSettings.vue [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+					@update:server="debounceUpdateServers"
+					@update:verify="debounceUpdateServers" />
+			</transition-group>
+		</ul>
+
+		<div class="signaling-secret">
+			<h4>{{ t('spreed', 'Shared secret') }}</h4>
+			<input type="text" name="signaling_secret" :disabled="loading"
+				:placeholder="t('spreed', 'Shared secret')" :value="secret"
+				:aria-label="t('spreed', 'Shared secret')" @update="debounceUpdateServers">
```
Comment (nickvergessen) [inline vue/src/SignalingServerSettings.vue]: `debounceUpdateServers` seems to not trigger :thinking:

File: vue/src/components/SignalingServer.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<template>
+	<div class="signaling-server">
+		<input ref="signaling_server" type="text" name="signaling_server"
+			placeholder="wss://signaling.example.org" :value="server" :disabled="loading"
+			:aria-label="t('spreed', 'TURN server URL')" @input="updateServer">
+		<!--<input :id="'verify' + index" type="checkbox" :name="'verify' + index"-->
```
Comment (nickvergessen) [inline vue/src/components/SignalingServer.vue]: I need help here too

**Comment by skjnldsv** [comment]
Done!

**Comment by nickvergessen** [comment]
@Ivansss please review

**Pull request merged**